.config-schema.json

```diff
@@ -464,38 +464,50 @@
                 }
             }
         },
+        "checkCache": {
+            "type": "object",
+            "properties": {
+                "limit": {
+                    "description": "the size limit (in items) of the cache for Check (queries and iterators)",
+                    "type": "integer",
+                    "default": "10000",
+                    "x-env-variable": "OPENFGA_CHECK_CACHE_LIMIT"
+                }
+            }
+        },
         "checkIteratorCache": {
             "type": "object",
             "properties": {
                 "enabled": {
-                    "description": "enable caching of datastore iterators of Check requests",
+                    "description": "enable caching of datastore iterators. Each iterator is the result of a database query, for example usersets related to a specific object, or objects related to a specific user, up to a certain number of tuples per iterator. If the request's consistency is HIGHER_CONSISTENCY, this cache is not used.",
```

Comment:
@jpadilla could you tell me what to write here in terms of cache invalidation? I cannot tell from looking at the code: https://github.com/openfga/openfga/blob/f1f1b54b2163b4918a9509dbb5894df39293bd27/internal/graph/storagewrapper.go#L410-L412